entries/o/otpbanka.hr.json

```diff
@@ -1,11 +1,12 @@
 {
   "OTP banka": {
     "domain": "otpbanka.hr",
-    "url": "https://www.otpbanka.hr/",
     "tfa": [
-      "custom-software",
       "custom-hardware"
     ],
+    "custom-hardware": [
+      "OTP token device"
+    ],
```

Comment:
The device is described by themselves as an "OTP Token" so I went with their name for it.
It's a normal card-reading device that creates HOTP-codes using the bank card's chip and pin.